site.srht.shrimple.ouch.json

```diff
@@ -0,0 +1,90 @@
+{
+	"id": "site.srht.shrimple.ouch",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "47",
+	"sdk": "org.gnome.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.rust-stable"
+	],
+	"command": "ouch",
+	"finish-args": [
+		"--share=network",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--device=dri",
+		"--socket=wayland",
+		"--filesystem=home"
+	],
+	"build-options": {
+		"append-path": "/usr/lib/sdk/rust-stable/bin",
+		"build-args": [
+			"--share=network"
```

Comment:
this is not allowed the build must be done offline.

Reply:
so how should i build the package?

Reply:
Use a vendored tarball or use flatpak-cargo-generator to generate a dependency manifest (see dependency manifest section in requirements linked in PR)